src/serviceFacades/analyses.js

```diff
@@ -60,6 +60,19 @@ function getAnalysisRelaunchInfo(key, { id }) {
     });
 }
 
+function searchAnalysesInfinite(
+    key,
+    { rowsPerPage, orderBy, order, filter },
+    page = 0
+) {
+    return callApi({
+        endpoint: `/api/analyses?limit=${rowsPerPage}&sort-field=${orderBy}&sort-dir=${order.toUpperCase()}&offset=${
+            rowsPerPage * page
+        }&filter=[${filter}]`,
+        method: "GET",
+    });
+}
+
```

Comment:
`useInfiniteQuery` requires the last parameter to be the page number.

Reply:
Maybe could we add that as a comment?  I know it's something I'll at least forget